gog_galaxy

```diff
@@ -0,0 +1,17 @@
+[Section]
+Name = GOG Galaxy
+Version = 1.1.15.13
+License = Freeware
+Description =  The gaming Client designed for a convenient purchasing, playing and updating games, as well as an online play between gaming platforms, GOG Galaxy is also built with optionality in mind, and a belief that you should own the games you buy. DRM-free!
+Size = 139 MiB
+Category =  4
+URLSite = https://www.gog.com/galaxy
+URLDownload = https://web.archive.org/web/20160928035203if_/https://cdn.gog.com//open/galaxy/client/setup_galaxy_1.1.15.13.exe
```

Comment:
I'm not sure using that kind of links is the best. Is there any alternative?

Reply:
I do not know any other place  where we can download exactly this version

Reply:
Use that link instead: https://svn.reactos.org/packages/setup_galaxy_1.1.15.13.exe ;-)

Reply:
@HeisSpiter web.archive gave us permission to link to their downloads directly.

Reply:
Well, up to you.
It was in case they would delete it and because sometimes their archive is not that reliable (bad gateway and such when trying to download).